### DISCUSSION TITLE
V self-compiling compiler ?

### DISCUSSION THREAD

**Discussion by thi8v** [vlang/v#18444] [Questions and Answers] [answered]
Does V had a self-compiling compiler / or a compiler to c ?

Because I search in https://github.com/vlang/v and https://github.com/vlang/vc, at the first commits (around 100 commits) and I never found code that is not in V

**Comment by JalonSolov** [comment, marked as answer]
As of the first public version of V, yes, it compiles itself.

The default output is to C, then a C compiler turns that into an executable.

There are other types of output in various stages of completion - from most to least complete: JavaScript, WASM, Go, native.  A third-party has made a FreePascal backend as well.
